### DISCUSSION TITLE
How are these items grouped?

### DISCUSSION THREAD

**Discussion by DemonSlayer112** [tr7zw/ItemSwapper#135] [Q&A]
Hello!
Using KubeJS to make custom blocks and items and wondering how I can add things for my block variants if its even possible.
Also could i use this to make things craftable?
You see, i have 3+ variants for overlay textures, only difference being textures on blocks, so it would be great if i can in survival swap these even if their not in the inventory, would save me having to make a recipe for each variant.

**Comment by tr7zw** [comment]
> how I can add things for my block variants if its even possible.

If they have actual custom block ids, via the resource pack. If they are just vanilla blocks with texture ids, then it's not possible.

> Also could i use this to make things craftable?

This mod has nothing to do with crafting.

> so it would be great if i can in survival swap these even if their not in the inventory

Swapping to items not in the inventory is only working in creative mode. But also same limitation as above applies, they need to be actual custom block ids.
